CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Android: Refactored AdaptationConfig to enable the `onVideoAdaptation` callback
```

Comment:
I think this changelog isn't necessary, as the existing api's can be called the same way as before, or am I missing something?